.github/workflows/ci_clang_format_check.yml

```diff
@@ -0,0 +1,17 @@
+name: clang-format
+on: [push, pull_request]
+jobs:
+  formatting-check:
+    name: Clang Format Check
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        path:
+          - 'stuffer'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run clang-format style check for C/C++ programs.
+        uses: jidicula/clang-format-action@v4.9.0
+        with:
+          clang-format-version: '13'
```

Comment:
Why this particular github action? Are there other options, or is this pretty much the primary accepted Github solution?

And why version 13 instead of the latest?

Reply:
Thanks for pointing this out. There is a better option: https://github.com/marketplace/actions/c-c-linter. It comes from a more widely continued to package and had quite a few more interesting options.